### ISSUE TITLE
MAX - EXP - Object Library Issues

### ISSUE THREAD

**Issue by OldFlak** [TheGameCreators/GameGuruRepo#5474]
For some time the Object Library is in a bit of a mess.

Two issues that are getting long in the tooths are:
When selecting audio files none of the tiles have the 'audio type image' applied.
Example:
![image](https://github.com/TheGameCreators/GameGuruRepo/assets/12246318/62204060-0408-4cac-9800-e61290345910)

When selecting Objects the left tree does not size according to open nested folders correctly.
Example:
https://youtu.be/R93bOZ9PF1k

Thanks for checking :)

**Comment by LeeBamberTGC** [comment]
@OldFlak Fixed the missing sound icon (and other media types) for the next DEVTEST build.  I tried to reproduce the video issue of the tree not showing the end but could not reproduce. I expanded ALL the folders in my asset area, all the DLCs basically, and could always see the last sub folder which is WEAPONS:
![image](https://github.com/user-attachments/assets/27df34d0-c877-49a0-b193-fa8b99488a8f)

Try with the next DEVTEST build to see if this happens.  Thanks!
